frontend/pages/admin/OrgSettingsPage/cards/Advanced/Advanced.tests.tsx

```
@@ -38,24 +38,34 @@ describe("Advanced settings — Activity & data retention", () => {
     ).toBeInTheDocument();
   });
 
   it("starts with both checkboxes unchecked when collection is enabled", () => {
     renderAdvanced();
-    expect(screen.getByLabelText(/Disable hosts online/i)).not.toBeChecked();
-    expect(screen.getByLabelText(/Disable vulnerabilities/i)).not.toBeChecked();
+    expect(
+      screen.getByLabelText(/Disable hosts online historical reporting/i)
+    ).not.toBeChecked();
+    expect(
+      screen.getByLabelText(/Disable vulnerability exposure historical reporting/i)
```

ESLint

[error] Replace `/Disable·vulnerability·exposure·historical·reporting/i` with `⏎········/Disable·vulnerability·exposure·historical·reporting/i⏎······`

(prettier/prettier)